packages/app/app/components/PlayQueue/index.js

```diff
@@ -47,7 +54,7 @@ class PlayQueue extends React.Component {
 
     return this.props.items.map((el, i) => {
       return (
-        <Draggable key={i} index={i} draggableId={i}>
+        <Draggable key={i} index={i} draggableId={`${i}`}>
```

Comment:
breaking change was that `draggableId` has to be a string now. 

I initially tried using `uuid` for the `key` and `draggableId`, but it was causing a different flickering issue. Plus, adding a playlist to the playqueue twice will cause repeats in `uuid`. Looking for any alternatives/suggestions.